packages/components/Organisms/Accordion/Accordion.vue

```diff
@@ -255,6 +255,7 @@ export default {
     }
 
     &__content {
+      max-width: $rpl-content-max-width;
```

Comment:
@rubyllamadora I think after you have the max width on `p` and headings in markup component, this will be unnecessary. Also this will add limit on all markup elements like table which doesn't need it. So just remove it should be ok. :)

Reply:
Thanks Tim. Will update